user-service/src/main/java/com/movie/recommender/user/controller/UserController.java

```diff
@@ -33,5 +33,30 @@ public ResponseEntity<String> login(@Valid @RequestBody UserLoginDTO userLoginDT
         String token = userService.login(userLoginDTO.getUsername(), userLoginDTO.getPassword());
         return ResponseEntity.ok(token);
     }
+
+    @GetMapping("/{userId}/favorites")
```

Comment:
keep `UserController` focused on user operations like authentication and profile management
create `FavoritesController` to manage favorite movies
move all favorite related methods to `FavoritesController`